src/components/organisms/UiSimpleQuestion/UiSimpleQuestion.vue

```diff
@@ -1,8 +1,24 @@
 <template>
   <div
     class="ui-simple-question"
-    role="radiogroup"
+    role="group"
+    aria-labelledby="ssn1"
   >
+    <slot
+      name="legend"
+      v-bind="{
+        legend,
+        id,
+      }"
+    >
+      <span
+        :id="id"
+        class="visual-hidden"
+        :aria-hidden="true"
```

Comment:
it doesn't make sense when you use it as a label and hidden it

Reply:
This is necessary.
Because VoiceOver reads this span as a legend for group. And if you remove aria-hidden VoiceOver reads again span before buttons. So please restore aria-hidden